src/ScheduleFunctions.cpp

```diff
@@ -1098,6 +1098,7 @@ void validate_schedule(Function f, Stmt s, const Target &target, bool is_output,
         }
     }
 
+#if 0
```

Comment:
#if 0'ed (or commented-out) code shouldn't be checked in. If this need to stay here-but-disabled, it needs a comment as to why.